infrastructure/private-api-config.yaml

```diff
@@ -0,0 +1,80 @@
+plugin: openapi
+specFile: ./private-api.yaml
+
+validation:
+  request: log
+
+resources:
+  - method: GET
+    path: /authorization
+    requestHeaders:
+      SessionHeader: session-id
+    response:
+      statusCode: 200
+
+  - method: GET
+    path: /authorization
+    requestHeaders:
+      SessionHeader: ""
+    response:
+      statusCode: 400
+
+  - method: GET
+    path: /authorization
+    requestHeaders:
```

Comment:
This doesn't need `requestHeaders` - if we want this to be the default behaviour if the 200 and 400 criteria aren't matched then you can just keep it super simple